modules/search/pages/create-custom-token-filter.adoc

```diff
@@ -12,42 +13,49 @@ xref:customize-index.adoc#token-filters[Token filters] can improve your search r
 
 == Prerequisites 
 
-* You have the Search Service enabled on a node in your cluster.
-For more information about how to change Services on your cluster, see xref:cloud:clusters:modify-database.adoc[].
+* You have the Search Service enabled on a node in your operational cluster.
+For more information about how to change Services on your operational cluster, see xref:cloud:clusters:modify-database.adoc[].
  
 * You have logged in to the Couchbase {page-ui-name}. 
 
-* You have started to create or already created an index in xref:create-search-index-ui.adoc[Advanced Mode].
+* You have started to create or already created an index in xref:create-search-indexes.adoc#advanced-mode[Advanced Mode Editing].
+
+* You have already created or started to create a xref:create-custom-analyzer.adoc[custom analyzer] in your Search index.
 
 == Procedure 
 
 To create a custom token filter with the {page-ui-name} in Advanced Mode: 
 
-. On the *Operational Clusters* page, select the cluster that has the Search index you want to edit. 
+. On the *Operational Clusters* page, select the operational cluster where you want to work with the Search Service. 
 . Go to menu:Data Tools[Search].
-. Click the index where you want to create a custom token filter.
-. Under *Advanced Settings*, expand *Custom Filters*. 
-+
-NOTE: Make sure you use *Advanced Mode*. 
-. Click btn:[Add Token Filter].
-. In the *Name* field, enter a name for the token filter.
+. Do one of the following:
+.. To work with an existing Search index, click the name of the index where you want to create a custom analyzer.
+.. To create a new Search index, click btn:[Create Search Index].
```

Comment:
This throws me off a bit as this seems to be listed as a prerequisite, but we're including the procedure? Might be a reason I'm unaware of, but I thought it was a copy and paste error.

Reply:
Might be bad c/p. Let me fix.